src/test/java/io/specto/hoverfly/junit/dsl/SingleQuoteHttpBodyConverterTest.java

```diff
@@ -0,0 +1,27 @@
+package io.specto.hoverfly.junit.dsl;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SingleQuoteHttpBodyConverterTest {
+
+    @Test
+    public void shouldConvertSingleQuotes() {
+        final SingleQuoteHttpBodyConverter singleQuoteHttpBodyConverter =
+            SingleQuoteHttpBodyConverter.jsonWithSingleQuotes("{'bookingId':'1'}");
+
+        assertThat(singleQuoteHttpBodyConverter.body())
+            .isEqualTo("{\"bookingId\":\"1\"}");
+    }
+
+    @Test
+    public void shouldSkipDoubleQuotes() {
+        final SingleQuoteHttpBodyConverter singleQuoteHttpBodyConverter =
+            SingleQuoteHttpBodyConverter.jsonWithSingleQuotes("{\"bookingId\":\"1\"}");
```

Comment:
Shall we move this static factory method to `HttpBodyConverter`, so that users can see all available converters from one place?